crates/iota-indexer/src/test_utils.rs

```diff
@@ -137,6 +138,19 @@ pub async fn start_test_indexer_impl(
                 .await
             })
         }
+        IndexerTypeConfig::AnalyticalWorker => {
+            let blocking_pool =
+                new_connection_pool_with_config(&db_url, Some(5), Default::default()).unwrap();
```

Comment:
just checking, no need to reset the db here?

Reply:
Hm there seems to be some inconsistency for the existing worker already, not sure what's the correct way here.
The config we use it's hardcoded to false, but in this function something gets reset independent of the config.
I'll just add it, in case it will be used somewhere else

Reply:
Opened https://github.com/iotaledger/iota/issues/5365 to have this cleared up